src/briefcase/platforms/linux/appimage.py

```diff
@@ -27,6 +27,10 @@ class LinuxAppImagePassiveMixin(LinuxMixin):
     # docker exists. It is used by commands that are "passive" from the
     # perspective of the build system, like open and run.
     output_format = "appimage"
+    supported_host_os = {"Darwin", "Linux"}
+    supported_host_os_reason = (
+        "AppImages can only be built on Linux or in Docker on macOS."
```

Comment:
I found this message a little difficult to parse; I'd suggest: 
```suggestion
        "Linux Appimages can only be built on Linux, or on macOS using Docker."
```

Reply:
Updated. Although, I capitalized the `i` in `AppImages` and omitted the comma since it's technically a comma splice :) but I see the distinction it's trying to make.